.github/workflows/pull-request.yml

```
@@ -449,3 +449,153 @@
+  
+  generate-docs:
+    name: Generate Auto Documentation
+    runs-on: ubuntu-latest
+    needs: [Code-Quality-Checks, Test-Application, Start-App-Without-Docker, Docker-Start-Check]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0  # Required for getting full git history
+          PAT: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

actionlint (1.7.4)

input "PAT" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)